benefits/eligibility/templates/eligibility/includes/selection-label--veteran.html

```diff
@@ -7,8 +7,11 @@
 
 {% block description %}
   {% blocktranslate trimmed %}
-    This option is for people who have served in the active military, naval, or air service, and who were discharged or
-    released therefrom under conditions other than dishonorable. You will need to
-    <a href="https://www.va.gov/resources/verifying-your-identity-on-vagov/" target="_blank" rel="noopener noreferrer">verify your identity through VA.gov</a>.
+    This option is for people who have served in the active military, naval, or air service, and who were discharged or released therefrom under conditions other than dishonorable. You will need to verify your identity with
   {% endblocktranslate %}
+
+  {% include "core/includes/modal-trigger.html" with classes="border-0 bg-transparent p-0 login" modal="modal--login-gov-veterans" login=True period=True %}
```

Comment:
Quibble: `veteran` should be singular in the modal ID here and below, to match the include naming format.